resources/lib/vrtplayer.py

```diff
@@ -110,7 +110,7 @@ def show_channels_menu_items(self, channel=None):
             self._kodi.show_listing(tvshow_items, sort='label', content='tvshows')
         else:
             from resources.lib import CHANNELS
-            channel_items = self._apihelper.get_channel_items(action=actions.LISTING_CHANNELS, channels=[c.get('name') for c in CHANNELS])
+            channel_items = self._apihelper.get_channel_items(action=actions.LISTING_CHANNELS, channels=[c.get('name') for c in CHANNELS if self._kodi.get_setting(c.get('name')) == 'true'])
```

Comment:
This was not the intention. The channel selection was purely for filtering "Most recent" and "Soon offline".

Reply:
It's okay for me to adapt this, it just seemed a logical thing to do.

Reply:
I didn't even consider this. But I am open to change my mind.